tools/svaba.cwl

```diff
@@ -1,53 +1,88 @@
+  normal_bams:
+    type:
+      - 'null'
+      - type: array
+        items: File
+        inputBinding:
+          prefix: '--normal-bam'
+    secondaryFiles: [{pattern: '^.bai', required: true}]
+    inputBinding:
+      position: 0
+    doc: "Control BAM/CRAM/SAM file (eg normal). Can input multiple. Optional."
+  reference_genome: { type: 'File', secondaryFiles: [{pattern: '.fai', required: true}, {pattern: '.64.amb', required: true}, {pattern: '.64.ann', required: true}, {pattern: '.64.bwt', required: true}, {pattern: '.64.pac', required: true}, {pattern: '.64.sa', required: true}], inputBinding: { prefix: '--reference-genome', position: 0 }, doc: "Path to indexed reference genome to be used by BWA-MEM." }
 
   dbsnp_vcf: { type: 'File?', inputBinding: { prefix: '--dbsnp-vcf', position: 0 }, doc: "DBsnp database (VCF) to compare indels against" }
   region_file: { type: 'File?', inputBinding: { prefix: '--region-file', position: 0 }, doc: "Run on targeted intervals. Accepts BED file" }
   blacklist: { type: 'File?', inputBinding: { prefix: '--blacklist', position: 0 }, doc: "BED-file with blacklisted regions to not extract any reads from." }
   germline_sv_database: { type: 'File?', inputBinding: { prefix: '--germline-sv-database', position: 0 }, doc: "BED file containing sites of known germline SVs. Used as additional filter for somatic SV detection." }
 
-  gzip: { type: 'boolean?', default: true, inputBinding: { prefix: '--g-zip', position: 0 }, doc: "Gzip and tabix the output VCF files." }
-  germline: { type: 'boolean?', default: false, inputBinding: { prefix: '--germline', position: 0}, doc: "Sets recommended settings for case-only analysis (eg germline). (-I, -L5, assembles NM >= 3 reads)" }
-  rules: { type: 'boolean?', default: false, inputBinding: { valueFrom: "$(self ? '--rules all' : '')", position: 0 }, doc: "Default behavior is just assemble clipped/discordant/unmapped/gapped reads. Override?" }
+  germline: { type: 'boolean?', inputBinding: { prefix: '--germline', position: 0}, doc: "Sets recommended settings for case-only analysis (eg germline). (-I, -L5, assembles NM >= 3 reads)" }
+  rules: { type: 'boolean?', inputBinding: { valueFrom: "$(self ? '--rules all' : '')", position: 0 }, doc: "Default behavior is just assemble clipped/discordant/unmapped/gapped reads. Override?" }
   highly_parallel: { type: 'boolean?', default: false, inputBinding: { prefix: '--hp', position: 0 }, doc: "Highly parallel. Don't write output until completely done. More memory, but avoids all thread-locks." }
-  no_interchrom_lookup: { type: 'boolean?', default: false, inputBinding: { prefix: '--no-interchrom-lookup', position: 0 }, doc: "Set true to not do mate-region lookup if mates are mapped to different chromosome." }
+  no_interchrom_lookup: { type: 'boolean?', inputBinding: { prefix: '--no-interchrom-lookup', position: 0 }, doc: "Set true to not do mate-region lookup if mates are mapped to different chromosome." }
```

Comment:
Why remove the default here?

Reply:
Because its an optional input. If no input is provided it is functionally equivalent to having a default of false. So mostly just to remove redundancy. I do notice I didn't remove that for highly_parallel though so I'll do that to make them the same.